src/fast_api.rs

```diff
@@ -148,6 +157,36 @@ struct CTypeSequenceInfo {
   sequence_type: SequenceType,
 }
 
+#[repr(C)]
+pub union FastApiCallbackData {
+  /// `data_ptr` allows for default constructing FastApiCallbackOptions.
+  pub data_ptr: *mut c_void,
+  /// The `data` passed to the FunctionTemplate constructor, or `undefined`.
+  pub data: crate::Value,
+}
+
+/// A struct which may be passed to a fast call callback, like so
+/// ```c
+/// void FastMethodWithOptions(int param, FastApiCallbackOptions& options);
+/// ```
+#[repr(C)]
+
+pub struct FastApiCallbackOptions {
```

Comment:
I believe V8 has an API to create a dummy `FastApiCallbackOptions` object for testing. It would perhaps be good to create a size assertion test with that, somehow.